ports/hordes/The Hordes.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/hordes"
+
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$LD_LIBRARY_PATH"
+
+cd $GAMEDIR
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Rename data.win
+[ -f "./data.win" ] && mv data.win game.droid
+
+$ESUDO chmod 666 /dev/uinput
+
+$GPTOKEYB "gmloadernext" &
+
+$GPTOKEYB "gmloadernext" -c "hordes.gptk" &
```

Comment:
Does this work? It usually looks like this: -c "./portname.gptkS" &

Reply:
> Does this work? It usually looks like this: -c "./portname.gptk" &

Yep, it looks for that in the current directory since we already did `cd $GAMEDIR`. The `./` is redundant.